.travis.yml

```diff
@@ -41,4 +41,4 @@ script:
   - travis_run_tests
 
 after_success:
-  - travis_after_tests_successs
+  - travis_after_tests_success
```

Comment:
You didn't rebase onto 9.0 like I asked. Please do. This is in the diff & it should not be